core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1697,4 +1706,54 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		return $label;
 	}
+
+	/**
+	 * @param array<int|string> $ids
+	 */
+	protected function preloadCurrentRecords(array $ids, string $table): void
```

Comment:
Why is this in the generic `DataContainer` class, which is not necessarily related to tables? Shouldn't this `abstract` here an implemented in `DC_Table`?